### DISCUSSION TITLE
Is it possible to attach to a currently running process to get its log?

### DISCUSSION THREAD

**Discussion by SergeyBibikov** [doronz88/pymobiledevice3#886] [Q&A]
Hi! 
I launch the application using the multiprocessing.Process and get the application logs doing the following:
``` 
  lockdown = create_using_usbmux(serial=device_id)
  with DvtSecureSocketProxyService(lockdown=lockdown) as dvt:
        process_control = ProcessControl(dvt)
        pid = process_control.launch(
            bundle_id=bundle_id,
            kill_existing=True,
            start_suspended=False
        )
        logging.info(f'Process launched with pid {pid}')
        while True:
            for output_received in process_control:
                logging.info(output_received.message.strip()+'\n')
```
For some reason the multiprocessing.Process exits(maybe the `dvt` is closed, but I'm not sure)
I would like to handle this situation and "attach" to the application process running on the device without relaunching the app, is it possible? At least, in theory :)

Thanks!

**Comment by doronz88** [comment]
If the logging is done via the logging API, isn't `pymobiledevice3 syslog live` sufficient?

**Reply by SergeyBibikov** [reply]
The logs in question don't get into the syslog, unfortunately. It's primarily some info/debug messages from swift files.
When I use the libimobiledevice library, I just redirect the output of it's `idevicedebug -d run` command to a file
All in all the solution with the DvtSecureSocketProxyService works too but if the multiprocessing.Process terminates while the app is still running, I'd like to reattach to the app process.
